kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-planning/src/test/java/org/kie/server/services/taskassigning/planning/SolverHandlerTest.java

```diff
@@ -125,6 +149,62 @@ public void setUp() {
         doReturn(solutionProcessor).when(handler).createSolutionProcessor(eq(delegate), any(), eq(TARGET_USER), anyInt());
     }
 
+    @After
+    public void cleanUp() {
+        System.clearProperty(TASK_ASSIGNING_PROCESS_RUNTIME_TARGET_USER);
+        System.clearProperty(TASK_ASSIGNING_PUBLISH_WINDOW_SIZE);
+        System.clearProperty(TASK_ASSIGNING_SYNC_INTERVAL);
+        System.clearProperty(TASK_ASSIGNING_SYNC_QUERIES_SHIFT);
+        System.clearProperty(TASK_ASSIGNING_USERS_SYNC_INTERVAL);
+        System.clearProperty(TASK_ASSIGNING_FIRST_LEVEL_LOCAL_SEARCH_TIME);
+        System.clearProperty(TASK_ASSIGNING_SECOND_LEVEL_LOCAL_SEARCH_TIME);
+    }
+
+    @Test
+    public void getTargetUserId() {
+        assertThat(TARGET_USER).isEqualTo(handler.getTargetUserId());
```

Comment:
If the goal is to test the propagation of the parameters to the `SolverHandler`, the assertion should be the other way around:
`assertThat(handler.getTargetUserId()).isEqualTo(TARGET_USER);`

Please see the other tests too.

Reply:
damn, good catch.